packages/flutter_tools/lib/src/run.dart

```diff
@@ -198,8 +197,6 @@ class RunAndStayResident extends ResidentRunner {
     }
 
     printStatus('Application running.');
-    if (debuggingOptions.buildMode == BuildMode.release)
```

Comment:
I'd like to land this as is; I'd like to avoid having checks and different behavior based on modes.